std/regex/package.d

```diff
@@ -80,9 +80,17 @@ $(TR $(TD Objects) $(TD
   assert(m.front[1] == "12");
   ...
 
-  // The result of the `matchAll/matchFirst` is directly testable with if/assert/while.
-  // e.g. test if a string consists of letters:
-  assert(matchFirst("Letter", `^\p{L}+$`));
+  // The result of `matchAll/matchFirst` is directly testable with `if/assert/while`,
+  // e.g. test if a string consists of letters only:
+  assert(matchFirst("LettersOnly", `^\p{L}+$`));
+
+  // And we can take advantage of the ability to define a variable in the `IfCondition`:
```

Comment:
I would like `IfCondition` to be a hyperlink to https://dlang.org/spec/statement.html#if-statement. Could anyone show me how to achieve that, please ?

Reply:
You can do: $(LINK2 https://dlang.org/spec/statement.html#if-statement, `If Statement`). Also, there is a list of predefined macros for ddoc [here](https://dlang.org/spec/ddoc.html#predefined_macros)